### PR TITLE
IVS-178 - VER000

### PR DESCRIPTION
**Pull request by Ghesselink**
Simplified version of https://github.com/buildingSMART/ifc-gherkin-rules/pull/314

We concluded that the [concept template](https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/concepts/Object_Attributes/Revision_Control/content.html) does not provide enough context to set constraints on the sequence of given statements. The most simple version to colour blocks green should suffice.

### CODE REVIEW

File: features/VER000_Versioning-and-revision-control.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@implementer-agreement
+@VER
+@version1
+@E00020
+
+Feature: VER000 - Versioning and revision control
+    The rule verifies the capability to track changes to building data over time and maintain a comprehensive history of those changes.
```
Comment (evandroAlfieri) [inline features/VER000_Versioning-and-revision-control.feature:7]:
```suggestion
    The rule verifies the presence of IFC entities used to to track changes to building data over time and maintain a comprehensive history of those changes.
```

**Review by evandroAlfieri — CHANGES_REQUESTED** [review]

File: features/VER000_Versioning-and-revision-control.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@implementer-agreement
+@VER
+@version1
+@E00020
+
+Feature: VER000 - Versioning and revision control
+    The rule verifies the presence of IFC entities used to to track changes to building data over time and maintain a comprehensive history of those changes.
```
Comment (civilx64) [inline features/VER000_Versioning-and-revision-control.feature:7]:
```suggestion
    The rule verifies the presence of IFC entities used to track changes to building data over time and maintain a comprehensive history of those changes.
```

File: test/files/ver000/pass-ver000-owner_wrong_change_action.ifc
Comment (civilx64) [inline test/files/ver000/pass-ver000-owner_wrong_change_action.ifc:1]: The name of this file indicates it relates to a specific case from the previous PR and is no longer needed.  I recommend deleting it unless there is a specific need to keep it.

**Review by civilx64 — COMMENTED** [review]
The conversion-based unit for .DEGREE. was triggering a schema error so I removed it from the test files in ad84815b8fc00b7e89537bd9b98459431a8c2e68.  Small typo fix required in the `.feature` file and then we are good for UAT.

**Review by civilx64 — APPROVED** [review]

File: features/VER000_Versioning-and-revision-control.feature [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@implementer-agreement
+@VER
+@version1
+@E00020
+
+Feature: VER000 - Versioning and revision control
+    The rule verifies the presence of IFC entities used to track changes to building data over time and maintain a comprehensive history of those changes.
```
[the remark this answers is not in the document]
Comment (Ghesselink) [reply features/VER000_Versioning-and-revision-control.feature:7]: We can still modify it. What about 
 >  The rule verifies the presence of IFC entities used to track changes to building data over time, providing information on the latest modifications, including when and by whom they were made.

[the remark this answers is not in the document]
Comment (Ghesselink) [reply features/VER000_Versioning-and-revision-control.feature:7]: >  but I'm not sure whether it's a well-understood word.

Well, I had to look it up myself :) @civilx64 ,as our only native English speaker of the team, what do you think? 

I would opt for the second sentence; it's clear, and consistency between the docs and validation service is also nice.

**Pull request merged**